integreat_cms/static/src/js/media-management/component/upload-file.tsx

```diff
@@ -24,73 +28,46 @@ export default function UploadFile({
   mediaTranslations,
   submitForm,
   isLoading,
+  refreshListing,
 }: Props) {
   // This state is a buffer which contains the currently selected file (the browser type File)
   const [selectedFile, setSelectedFile] = useState<File | null>(null);
+  const dropZoneRef = useRef();
+
+  useEffect(() => {
+    const dropZone = new Dropzone(dropZoneRef.current, {
+      url: apiEndpoints.uploadFile,
+      headers: { "X-CSRFToken": getCsrfToken() },
+      dictDefaultMessage: mediaTranslations.text_upload_area,
+      acceptedFiles: allowedMediaTypes,
+    });
+    dropZone.on("queuecomplete", () => {
+      refreshListing();
+      dropZone.removeAllFiles(true);
+    });
+    dropZone.on("complete", () => {
+      refreshListing();
```

Comment:
I don't really understand why, but this does not seem to work for me.
When I upload files, only sometimes the new file is listed in the library and sometimes I need to reload the page to see the files.